src/db/models/mint_config_tx.rs

```diff
@@ -166,6 +166,17 @@ impl MintConfigTx {
             .load::<i64>(conn)?;
         Ok(mint_amounts.into_iter().map(|val| val as u64).sum())
     }
+
+    /// Get mint config txs by block index
+    pub fn get_by_block_index(
+        block_index: BlockIndex,
+        conn: &Conn,
+    ) -> Result<Vec<MintConfigTx>, Error> {
+        Ok(mint_config_txs::table
+            .filter(mint_config_txs::block_index.eq(block_index as i64))
+            .order_by(mint_config_txs::block_index.desc())
```

Comment:
This is not going to do anything since you are only filtering for a single block index. However, it might be worth sorting by `id` (ascending) so that the order matches the order in which they were inserted.